Draft: Meta README (#4)
MeltanoLabs/Meta: README.md

We also encourage community members to use this as a place where existing connectors can be rebuilt or ported over to the Meltano SDK.

If you'd like to create a new connector in MeltanoLabs create an issue labeled "Add New Connector - [your connector name]" in the [Meta repo](https://github.com/MeltanoLabs/Meta) and assign it to @tayloramurphy and @pnadolny13 when youre code is ready to be pushed up to GitHub and they will create you a repository and give you access.
Alternatively you could build it in your own namespace and later migrate it once its ready.

### Migrating An Existing Connector

Its understandable that sometimes community members build new connector or create a fork and add features in their own namespace which then becomes the primary variant used by the community, but they no longer have the capacity or desire to maintain it.
In this case the primary maintainer has the option to migrate their connector to MeltanoLabs where it can live and be maintained into the future.

> Connectors in MeltanoLabs have quaility expectations, if the connectors isn't there yet we will label it with the appropriate maintenance status in its current state and request community contributions to get it up to the Active (Stable) status we expect.

Reviewer: ```suggestion
Connectors in MeltanoLabs have quality expectations. If the connector does not meet the standard then we will label it with the appropriate maintenance status in its current state and request community contributions to get it up to the Active (Stable) status we expect.
```

### Finding Maintainers

The initial approach will be to ask for volunteers within the MeltanoLabs community to take ownership of a respository as maintainer but we know that for some less frequently used connectors there might not be enough volunteers.

In this case we will do the following:
1. Whenever we receive a PR or Issue for a connector that has no maintainers, we announce this to the contributor.
1. We will post the PR or Issue to the MeltanoLabs contributors via the `labs` Slack channel to request community reviews. Known past contributors and maintainers will get mentioned.
1. If no one from the community responds in a timely manner, we can offer the PR author to become a contributor if they have a history of contributing to Meltano and/or other open source projects. (We can use their GitHub history to help in this evaluation process.)
1. Any successfully-merged PR automatically promotes that person to "contributor" and optionally to maintainer if they've consented in step 3 and no other maintainers exist.

## Maintenance Statuses

Active (Stable): The connector repository is well maintained and recommended for use in production environments.

> The repository has all of the following attributes; all known issues and bugs are addressed promptly, changes are expected to go through pull requests reviews, releases to main are created on the appropriate cadence, security patches are addressed promptly, a CI pipeline that runs automated testing is present, community contributed pull requests are reviewed in a timely manner, README is complete with examples and detail, etc.

Reviewer: ```suggestion
The repository has all of the following attributes; all known issues and bugs are addressed promptly, changes are expected to go through pull request reviews, releases to main are created on the appropriate cadence, security patches are addressed promptly, a CI pipeline that runs automated testing is present, community contributed pull requests are reviewed in a timely manner, README is complete with examples and detail, etc.
```
